package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "openmrs-module-appointments-frontend",
-    "version": "0.0.2",
+    "version": "0.0.3",
```

Comment:
@binduak @buvaneswari-arun, are you clear on this version upgrade? If yes, can the PR be merged?